HW_1/Task_1.py

```diff
@@ -0,0 +1,21 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+username = input('Введите ваше имя: ')
+
+x = int(input('Введите трехзначное число: '))
+a = int(x//100)
```

Comment:
Целочисленное отделение и так возвращает тип данных `int`.
```suggestion
a = x // 100
```